content/resources.md

```diff
@@ -160,6 +160,34 @@ Below is the JSON standard format representing a set of categories.
 ::: panel-link Want more details about the category resource? [Check its endpoints here!](/api-reference.html#Categories)
 :::
 
+## Association type (2.0 only)
+
+To facilitate the work of Julia inside the PIM, we gather attributes into groups. These groups are called `Attribute groups`.
```

Comment:
attribute group?

Reply:
It's association type chapter, so I think the whole sentence is wrong, no?

Reply:
Yep, that's a wron copy-paste. My bad, will correct it.